doc/src/guides/solving/index.md

```diff
@@ -16,16 +16,17 @@ Learn how to use SymPy computer algebra system to:
 | [ Solve a system of equations (linear or nonlinear) algebraically ](solve-system-of-equations-algebraically.md)              | $x^2 + y = 2z, y = -4z$ | $\{(x = -\sqrt{6z}, y = -4z),$ ${(x = \sqrt{6z}, y = -4z)\}}$                                                                                        |
 |  [Solve one or a system of equations numerically](solve-numerically.md)                           | $\cos(x) = x $ | $ x \approx 0.739085133215161$                                                                                           |
 |  {func}`Solve an ordinary differential equation algebraically <sympy.solvers.ode.dsolve>`   | $y''(x) + 9y(x)=0 $ | $ y(x)=C_{1} \sin(3x)+ C_{2} \cos(3x)$                                                    |
+| [ Solve (find the roots of) a polynomial algebraically or numerically ](solve-polynomial.md)                       | $ ax^2 + bx + c = 0 $ | $ x = \frac{-b\pm\sqrt{b^2 - 4ac}}{2a} $                                                                                                |
 |  [ Solve a matrix equation algebraically ](solve-matrix-equation.md)                    | $ \left[\begin{array}{cc} c & d\\1 & -e\end{array}\right] \left[\begin{array}{cc} x\\y\end{array}\right] = \left[\begin{array}{cc} 2\\0\end{array}\right] $ | $ \left[\begin{array}{cc} x\\y\end{array}\right] = \left[\begin{array}{cc} \frac{2e}{ce+d}\\\frac{2}{ce+d}\end{array}\right]$  |
 |  [ Reduce one or a system of inequalities for a single variable algebraically ](reduce-inequalities-algebraically.md)           | $ x^2 < \pi, x > 0 $ | $ 0 < x < \sqrt{\pi} $                                                                                                  |
 | [ Solve (find the roots of) a polynomial algebraically ](../../modules/polys/basics.rst)                       | $ x^2 - x = 0 $ | $ x \in \{0, 1\} $                                                                                                |
```

Comment:
Should this entry be updated (or removed)?

Reply:
Also let's make sure the order in the table corresponds as closely as possible to the order in the sidebar.

Reply:
> Should this entry be updated (or removed)?

I don't think so--it links to the subject new polynomial solving page, which has algebraic and numeric methods. Perhaps I'm misunderstanding the basis of your question?

Reply:
Oh, I get your point--there are two polynomial entries. I deleted the second one.